### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by ShiroYaksha90**
## Hi 👋, in this pull request I:
-  Create a method called `current_user` in the ApplicationController to make current user data available to all controllers.
    -  It will return the first user from the database.
- Create forms to perform the following functions:
    - Creates a Post on behalf of the `current_user`.
    - Create a comment on behalf of the `current_user`.
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/views/posts/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  </div>
+
+  <div>
+    <%= form.label :text %>
+    <%= form.text_area :text %>
+  </div>
+
+  <div>
+    <%= form.submit "Create Post" %>
+  </div>
```
Comment (danmainah) [inline app/views/posts/new.html.erb:16]: - [ ] @ShiroYaksha90 Great work on your app, however, I was unable to test if the forms are working correctly, I got this error when trying to access the form page to create a new post, kindly consider fixing the error and ensure all forms are working as expected (user can add a post comment and like a post)
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @ShiroYaksha90 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Created a current_user method in your ApplicationController to make current user data available to all controllers
✔️ Added method to create a post comment and like a post.




# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by ShiroYaksha90** [comment]
@danmainah Thank you for the insightful review I'll implement the requested changes

**Review by sumairq — APPROVED** [review]
### Hi @ShiroYaksha90 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Forms are working properly
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**